src/hooks/useUserGauge.ts

```diff
@@ -57,17 +75,21 @@ export default function useUserGauge(gaugeAddress?: string): UserGauge | null {
     userGaugeRewards?.claimableExternalRewards.length,
   )
 
-  const boost =
-    gauge.gaugeBalance?.gt(Zero) && gauge.workingBalances?.gt(Zero)
-      ? `${formatUnits(
-          gauge.workingBalances
-            .mul(BN_1E18)
-            .div(gauge.gaugeBalance)
-            .mul(BigNumber.from(25))
-            .div(BigNumber.from(10)),
-          18,
-        )}x`
-      : "-"
+  const userLPAmount = gauge.gaugeBalance
```

Comment:
I think you can destructure this to be
`const { 
    userLpAmount: gaugeBalance,
    totalLpAmount: gaugeTotalSupply,
    workingBalances,
    workingSupply,
  } = gauge`